zipkin4net/Criteo.Profiling.Tracing/Criteo.Profiling.Tracing.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
From my understanding, xproj files shouldn't be commited. Did I miss something ?
If I'm right then this comment applies to every xproj files commited.